src/staging/staging_main.py

```diff
@@ -61,36 +61,33 @@ def run_staging(  # noqa: C901
     network_or_hdfs = config["global"]["network_or_hdfs"]
     is_network = network_or_hdfs == "network"
     load_from_feather = config["global"]["load_from_feather"]
-    load_updated_snapshot = config["global"]["load_updated_snapshot"]
-
+    # load_updated_snapshot = config["global"]["load_updated_snapshot"]
 
     # set up dictionaries with all the paths needed for the staging module
     staging_dict = config["staging_paths"]
 
-    use_frozen_snapshot_data = (not config["global"]["run_first_snapshot_of_results"])
-    if not use_frozen_snapshot_data:
-
-        snapshot_name = os.path.basename(staging_dict["frozen_snapshot_path"]).split(
-            ".", 1
-        )[0]
+    stage_frozen_snapshot = config["global"]["run_first_snapshot_of_results"]
+    stage_updated_snapshot = config["global"]["load_updated_snapshot_for_comparison"]
+    if stage_frozen_snapshot or stage_updated_snapshot:
         feather_path = staging_dict["feather_output"]
-        feather_file = os.path.join(feather_path, f"{snapshot_name}.feather")
 
-        if load_updated_snapshot:
-            secondary_snapshot_name = os.path.basename(
-                staging_dict["secondary_snapshot_path"]
+        if stage_frozen_snapshot:
+            snapshot_name = os.path.basename(staging_dict["frozen_snapshot_path"]).split(
+                ".", 1
+            )[0]
+
+            feather_file = os.path.join(feather_path, f"{snapshot_name}.feather")
+
+        elif stage_updated_snapshot:
+            updated_snapshot_name = os.path.basename(
+                staging_dict["updated_snapshot_path"]
             ).split(".", 1)[0]
-            secondary_feather_file = os.path.join(
-                feather_path, f"{secondary_snapshot_name}.feather"
+            feather_file = os.path.join(
+                feather_path, f"{updated_snapshot_name}.feather"
             )
-        else:
-            secondary_feather_file = None
 
-        # Check if the if the snapshot feather and optionally the secondary
-        # snapshot feather exist
-        feather_files_exist = helpers.check_snapshot_feather_exists(
-            config, check_file_exists, feather_file, secondary_feather_file
-        )
+        # Check if the if the snapshot feather exists
+        feather_files_exist = check_file_exists(feather_file)
```

Comment:
do we want this to use rd_file_exists now?

Reply:
pipeline is passing rd_file_exists to staging, but I've changed it's name in the module for consistency